packages/crew_url_opener.rb

```diff
@@ -0,0 +1,104 @@
+require 'package'
+
+class Crew_url_handler < Package
```

Comment:
`Crew_url_handler` class doesn't match the filename `crew_url_opener.rb`?

Reply:
Corrected to `crew_url_opener`